packages/telegram-cloud-storage-stamper/README.md

```diff
@@ -0,0 +1,195 @@
+
+### Preqrequisites 
+Telegram Bot API >= 6.9
+
+The Telegram Cloud Storage Stamper has a few different modes of operation, namely a classic [stamper](https://docs.turnkey.com/api-overview/stamps) for stamping requests made to Turnkey's API, and an interface for a Telegram Mini App built with Turnkey to interact with Telegram Cloud Storage. This provides the developer of the application utilities such as creating stamps on requests made by users, storing user API keys, storing temporary keys that are needed for decrypting credential bundles for activites like [email auth](https://docs.turnkey.com/features/email-auth) or [oauth](https://docs.turnkey.com/features/oauth), or storing arbitrary values that would be helpful to have saved for a user from session to session on device to device.
+
+The Telegram Cloud Storage Stamper will, by default, store the API key used for signing in Telegram Cloud Storage under the key `TURNKEY_API_KEY`. A Cloud Storage "key" is the index under which a value is stored in Telegram Cloud Storage. This can be changed when using the `.create()` or `.setSigningKey()` functions. An API key is stored within Cloud Storage as a JSON string of the following object:
+```
+{
+  apiPublicKey: "compressedApiPublicKeyHex",
+  apiPrivateKey: "apiPrivateKeyHex",
+}
+```
+
+#### Argument Usage
+
+The `.create()` and `.setSigningKey()` functions take one of the following 4 sets of arguments:
+- No arguments: Use an API key at the default location within Telegram Cloud Storage `TURNKEY_API_KEY` and set that as the signing key
+- Just an API key: Store the passed in API key and at the default Telegram Cloud Storage location and set that as the signing key
+- Just a Cloud Storage key: Use an API key stored at the specified Telegram Cloud Storage key location and set that as the signing key
+- Both an API key and a Cloud Storage key: Store the passed API key at the specified Telegram Cloud Storage key location and set that as the signing key
+  
+The `.getAPIKey()` and `.setAPIKey()` functions operate in a similar manner taking an optional `key` parameter that will be used to `get` or `set` the API key at that location if it is passed, or at the default location if it is not passed.
+
+### Usage
+Insert a new api key into Telegram Cloud Storage at the default API key location
+
+```ts
+import { TelegramCloudStorageStamper } from "@turnkey/telegram-cloud-storage-stamper";
+import { TurnkeyBrowserClient, TurnkeySDKClientConfig } from "@turnkey/sdk-browser";
+
+// create a new Telegram Cloud Storage Stamper
+const stamper = await TelegramCloudStorageStamper.create({
+  apiPublicKey: "...",
+  apiPrivateKey: "...",
+});
+
+if (!stamper) {
+  // There was a problem initializing the stamper
+}
```

Comment:
Is this actually true? I'd expect an error thrown if the stamper couldn't be initialized (instead of a null/undefined/falsy stamper)

Reply:
changed this to throw an error instead of returning undefined